sample.env

```diff
@@ -4,5 +4,8 @@ JWT_SECRET = 'secretkey4u'
 JWT_ADMIN_SECRET = 'secretkey4adminAPI_KEY'
 PORT = 5000
 SENDGRID_API_KEYS=''
+##Google Authentication
+CLIENT_ID=
```

Comment:
Describe the auth keys more specifically, e.g GOOGLE_CLIENT_ID, GOOGLE_CLIENT_SECRET

Reply:
Since that is the only issue can you just merge while I update my base then fix it on my end and send the PR for this change and possibly another one